### DISCUSSION TITLE
Margin doesn't work for svg icon

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8209] [Help]
I wanted to add some space between the svg icon and the text next to it using margin utilitiy classes, but it didn't work. I tried adding `mr-10` class to the `<svg>` tag and its parent div
demo: https://play.tailwindcss.com/QyB1S8ohyQ
Does tailwind treat placeholder text differently than normal text?

**Comment by hellomrbigshot** [comment]
Maybe it is caused by the 'absolute' class, you can use flex to replace it.
